gui/choc_WebView.h

```diff
@@ -125,6 +125,10 @@ class WebView
         /// Option to enable or disable the ability to scroll with the space key
         bool enableSpaceToScroll = true;
 
+        /// Option to enable or disable the ability to use backspace to navigate to the
+        /// previous page
+        bool enableBackspaceToNavigate = true;
```

Comment:
Should we call this `ignoreBackspace` instead? I believe "to navigate" is accurate, but it's arguably more detail that we need for the sake of this flag.

Reply:
No preference for me

Reply:
prefer the verbosity!